cysync/src/pages/mainApp/sidebar/settings/tabViews/deviceHealth/deviceUpgrade.tsx

```diff
@@ -293,7 +293,7 @@ const DeviceUpgrade: React.FC<DeviceSettingItemProps> = ({
 
   useEffect(() => {
     if (isRefresh) {
-      handleRetry();
+      handleNext();
```

Comment:
isRefresh is not a state variable, so adding it use `useEffect` does not make sense.

Reply:
God what was I thinking!